features/inventory.feature

```diff
@@ -15,10 +15,16 @@ Scenario: Retrive an ProductInformation
     And I set the "prod_id" to "2"
     And I press the "retrieve" Button
     Then I should see "2" in the "new_qty" field
-    And I should see "2" in the "used_qty" field
-    And I should see "2" in the "open_boxed_qty" field
-    And I should see "5" in the "restock_level" field
-    And I should see "9" in the "restock_amt" field
+    AND I should see "2" in the "used_qty" field
```

Comment:
Please merge with the latest master branch first.

Reply:
done

Reply:
It seems the 'AND' still exists in branch bdd_delete, could you change them to 'And' instead?

Reply:
Addressed.